### DISCUSSION TITLE
Comparison to facerecognition

### DISCUSSION THREAD

**Discussion by turion** [nextcloud/recognize#93] [Q&A] [answered]
There is an app doing face recognition, https://github.com/matiasdelellis/facerecognition. How does this app relate to the other?

**Comment by bonswouar** [comment]
I'm not sure about rapidity & accuracy (@marcelklehr might know more about that), but main differences are that the other app need manual steps to install (I believe that's why @marcelklehr decided to include a simpler one in his project) & to manually assign/combine faces.
For now our version only look for similar faces than the ones from the Contact app - we will probably add an UI soon though (#76).

**Comment by turion** [comment]
Maybe it makes sense to document this somewhere in the readme?

**Comment by arch-user-france1** [comment]
_why not?_

**Comment by hillbicks** [comment]
Hey everyone,

sorry to bump this old thread, but since this one is basically unanswered I thought I ask here. 

I think it is a rather unfortunate situation, although I completely understand why:

We currently have the fantastic memories app which is enhanced by this app, recognize.

Then there is the nc photos app, which only supports the people feature from the face recognition app. 

So I would have to basically train and keep two data sets up to date, which isn't really ideal I think. 

Maybe there is a way to combine these two projects into one? Bundle ressources and all that. 

Any thoughts on this @marcelklehr?

EDIT: I just realized that this is the official nextcloud repo and found a [post ](https://github.com/matiasdelellis/facerecognition/issues/593)on the face regonition issue tracker, which talks about this already. I will ask the nc photos developer what his take on this is. 

EDIT2: And here is the open [issue ](https://gitlab.com/nkming2/nc-photos/-/issues/138) for nc photos switching to recognize.

**Reply by marcelklehr** [reply]
> Then there is the nc photos app, which only supports the people feature from the face recognition app.

Nextcloud photos also uses recognize, not the face recognition app.

**Reply by hillbicks** [reply]
Sorry, I should've been more clear. I was talking about the android app nc photos, not the app within nextcloud. Sorry about that. 

This is the app I'm talking about. 

https://gitlab.com/nkming2/nc-photos


But like I mentioned in the edit, the dev is already aware of this and wants to migrate to photos in NC25.

**Comment by marcelklehr** [comment, marked as answer]
> How does this app relate to the other?

The two apps use different technical approaches
